econet_hvac_furnace.yaml

```diff
@@ -29,6 +29,7 @@ econet:
             id(heat_stage).publish_state(x[26]);
             id(static_pressure).publish_state(((x[15] << 8) + x[16]) / 5280.0);
             id(return_air_temperature).publish_state(((x[50] << 8) + x[51]) / 10.0);
+            id(outdoor_air_temperature).publish_state(((x[52] << 8) + x[53])/10.0);
```

Comment:
Does this value only exist/get data if there is an ODU? If so, does the corresponding sensor need to somehow move to the ODU YAML?

Reply:
All of the furnaces have an input for an Outdoor Air Temp thermistor. Unsure on air handlers.  Seems this value can get updated in multiple ways, maybe

Reply:
Expanding on @justbruce comment - in my case this comes from a thermistor located on the heat pump outside, but it is read by the furnace hardware.